juniper-vpn.py

```diff
@@ -22,10 +22,10 @@
 import socket
 import netifaces
 import datetime
+import re
 
 debug = False
 
-ssl._create_default_https_context = ssl._create_unverified_context
```

Comment:
Probably yes. I have absolutely no idea why I need this. Happy to remove for merging.